src/style.css

```diff
@@ -1 +1,55 @@
-/* add styles here */
+/* #region page */
+.page__body {
+  margin: 0;
+}
+
+/* #end region page */
+
+/* #stars section  start */
+
+.stars-content {
+  height: 96px;
+  width: 100px;
+  margin: 0 auto;
+  display: flex;
+  align-content: baseline;
+  flex-wrap: wrap;
+}
+
+.stars {
+  width: 116px;
+  height: 16px;
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  margin: 0 4px 0 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
combine it using a comma
```suggestion
.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2),
.stars--3 .stars__star:nth-child(-n + 3) {
  background-image: url(images/star-active.svg);
}
...
```

Reply:
<img width="432" alt="image" src="https://github.com/mate-academy/layout_stars/assets/70220667/cbdd349e-279a-4c8b-8a5c-b03be0cfc630">